### PR TITLE
Pos container

### PR DESCRIPTION
**Pull request by PierrickBrun**

### CODE REVIEW

File: pos_container/static/src/js/container.js [hunk 1]
```diff
@@ -372,4 +401,19 @@ odoo.define('pos_container.container', function (require) {
...
+    screens.OrderWidget.include({
+        set_value: function(val) {
+            this._super(val);
+            var order = this.pos.get_order();
+            var oline = order.get_selected_orderline();
+            if (oline) {
+                var mode = this.numpad_state.get('mode');
+                if( mode === 'tare'){
+                    oline.set_tare_mode('MAN');
+                    oline.set_tare(val);
```
Comment (robinkeunen) [inline pos_container/static/src/js/container.js]: Ici, ne dois-tu pas faire un `set_quantity(gross_weight - tare)`?

File: pos_container/static/src/js/container.js [hunk 2]
```diff
@@ -92,10 +120,37 @@ odoo.define('pos_container.container', function (require) {
...
+                        error_body = except.arguments && except.arguments[0] || except.message || error_body;
+                    }
+                    self.gui.show_popup('error',{
+                        'title': _t('Error: Could not Save Changes'),
+                        'body': error_body,
+                    });
+                }
+            );
+        },
+        deleted_container: function(){
```
[the remark this answers is not in the document]
Comment (PierrickBrun) [reply pos_container/static/src/js/container.js]: I don't get what "from_pos" adds, this is indeed from pos if we are in this file ? Also note that this function is the "deleteD_container", which is called after it has effectively been done

File: pos_container/static/src/js/container.js (hunk 1 above)
Comment (PierrickBrun) [reply pos_container/static/src/js/container.js]: Hum c'est probable, tu peux faire le changement ? Comme je ne peux pas tester avec balance je fais un peu les choses à l'aveugle pour ça

Comment (robinkeunen) [reply pos_container/static/src/js/container.js]: Si tu démarres le pos en mode débug, tu peux simuler scan et balance :-)

File: pos_container/static/src/js/container.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (robinkeunen) [reply pos_container/static/src/js/container.js]: I meant to highlight that we are removing the container from the POS screen as opposite to deleting from the backend as you do above. But ok with the naming.

[the remark this answers is not in the document]
Comment (PierrickBrun) [reply pos_container/static/src/js/container.js]: I see, indeed, this is clearer than "deleted" which is often read as "delete". I prefer to keep it that way though, it's more consistent with odoo's choices

File: pos_container/static/src/js/container.js (hunk 1 above)
Comment (PierrickBrun) [reply pos_container/static/src/js/container.js]: done

File: pos_container/static/src/js/container.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (robinkeunen) [reply pos_container/static/src/js/container.js]: Ok then :-)

**Pull request merged**